crates/p2p/src/commands.rs

```diff
@@ -94,11 +93,11 @@ impl<DSP: Message + Clone> From<PublishMessage<DSP>> for GossipsubMsg<DSP> {
     }
 }
 
-impl<DSP: Message + Default + Clone> UnsignedPublishMessage<DSP> {
+impl UnsignedPublishMessage {
```

Comment:
😍

Reply:
yes a lot of `da<` delete around `<` vim motions.